inference-engine/src/transformations/src/transformations/common_optimizations/fq_mul_fusion.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/fq_mul_fusion.hpp"
+#include "transformations/utils/utils.hpp"
+
+#include <memory>
+#include <vector>
+
+#include <ngraph/opsets/opset4.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/rt_info.hpp>
+
+namespace {
+  std::pair<ngraph::Output<ngraph::Node>, ngraph::Output<ngraph::Node>>
+  get_adjusted_output_range(ngraph::Output<ngraph::Node> out_low,
+                            ngraph::Output<ngraph::Node> out_high,
+                            ngraph::Output<ngraph::Node> mul_constant) {
+    const auto mul_out_low = std::make_shared<ngraph::opset4::Multiply>(out_low, mul_constant);
+    const auto mul_out_high = std::make_shared<ngraph::opset4::Multiply>(out_high, mul_constant);
+
+    ngraph::OutputVector folded_out_low(1), folded_out_high(1);
+
+    if (!mul_out_low->constant_fold(folded_out_low, {out_low, mul_constant})) {
```

Comment:
@jane-intel if I understand right, we can get sub-graph where out_low or out_high are not constants. So in this case we don't have to throw an exception. We need to return multiply output.